opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -673,6 +676,15 @@ def test_resource_detector_entry_points_non_default(self):
         self.assertEqual(resource.attributes["a"], "b")
         self.assertEqual(resource.schema_url, "")
 
+    @patch.dict(
+        environ, {OTEL_EXPERIMENTAL_RESOURCE_DETECTORS: ""}, clear=True
```

Comment:
You are missing a test with `{OTEL_EXPERIMENTAL_RESOURCE_DETECTORS: "os"}`

Reply:
Good catch, added such a test